### PR TITLE
new tests

### PR DESCRIPTION
**Pull request by zinebidrissijanati**

### CODE REVIEW

**Comment by Zanzas** [comment]
For feature PR branch must be named following thi naming convention :

type/subeject_of_the_branch

Type can be one of this value : feat / test / refactor / style / docs / chore

**Comment by Zanzas** [comment]
Commits should follow this convention
https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit-message-format

File: test/functional/task.spec.js [hunk 1]
```diff
@@ -3,6 +3,7 @@
 const { test, trait} = use('Test/Suite')('Task')
 const Task = use('App/Models/Task')
 trait('Test/ApiClient')
+//add some changes here
```
Comment (Zanzas) [inline test/functional/task.spec.js]: good job 😄 !!

To have a more readable unit test please follow this : 

https://medium.com/@pjbgf/title-testing-code-ocd-and-the-aaa-pattern-df453975ab80

Comment (zinebidrissijanati) [reply test/functional/task.spec.js]: Okey!

**Pull request merged**